File: BookStore.DataAccess/Models/PrintingEditionsFilterModel/PrintingEditionsFilterModel.cs
using BookStore.DataAccess.Models.Base;
using System.Collections.Generic;
using static BookStore.DataAccess.Entities.Enums.Enums;
using static BookStore.DataAccess.Models.Enums.Enums;

namespace BookStore.DataAccess.Models.PrintingEditionsFilterModels
{
    public class PrintingEditionsFilterModel : BaseFilterModel
    {
        public List<TypePrintingEdition> Categories { get; set; }
        public Currencys Currency { get; set; }
        public PrintingEditionSortType SortType { get; set; }
        public decimal MinPrice { get; set; }
        public decimal MaxPrice { get; set; }
    }
}

File: StoreClient1/src/app/shared/models/PeintingEditions/printing-editions-filter-model.ts
import { PrintingEditionSortType } from 'src/app/shared/enums/printing-edition-sort-type';
import { TypePrintingEdition } from 'src/app/shared/enums/type-printing-edition';
import {BaseFilterModel} from '../Base/base-filter-model';

export class PrintingEditionsFilterModel extends BaseFilterModel {
  Categories: Array<TypePrintingEdition>;
  SortType: PrintingEditionSortType;
  MinPrice: number;
  MaxPrice: number;
}
